rent_golem/backend/rent_golem/clusters/tests.py

```diff
@@ -1,3 +1,225 @@
-from django.test import TestCase
+from typing import List, Optional, Tuple
+from django.shortcuts import reverse
 
-# Create your tests here.
+from rest_framework.test import APIClient, APITestCase, APIRequestFactory
+from rest_framework import status
+
+from .models import Cluster
+from accounts.models import User
+
+
+class ClusterViewSetTest(APITestCase):
+    def setUp(self) -> None:
+        self.factory = APIRequestFactory()
+        self.client = APIClient()
+        self.owner = User.objects.create_user(email="test@test.com", password="test1234")
+        self.insignificant_user = User.objects.create_user(email="test2@test.com", password="test1234")
+
+    def create_cluster_object(
+            self,
+            uuid: str = "dcec5482-59ef-4c0e-9ea4-4e5451c3cbda",
+            package_type: str = Cluster.Package.AUTOMATIC,
+            status: str = Cluster.Status.STARTING,
+            additional_params=None,
+            size: int = 5,
+            owner: User = None
+    ) -> Cluster:
+        if owner is None:
+            owner = User.objects.create_user(email="test@test.com", password="test1234")
+        return Cluster.objects.create(
+            uuid=uuid,
+            package_type=package_type,
+            status=status,
+            additional_params=dict() if additional_params is None else additional_params,
+            size=size,
+            owner=owner
+        )
```

Comment:
Calling this function two times with empty owner will result in error (same user created again).

Reply:
It's only for testing, I thought it can be this way. Should I change it?